products/extensions/b2b-components/employee-management/guides/b2b-roles.md

```diff
@@ -9,4 +9,4 @@ nav:
 
 Roles can be used to bind multiple permission to employees with contexts. Every employee can have one assigned role. Based on that role and the containing permission, the employee will get access to certain information and functionalities.
 
-The business partner has the opportunity to create a “default” role that will be selected by default, when creating a new employee.
+The business partner has the opportunity to create a **default** role that will be selected by default, when creating a new employee.
```

Comment:
Is this necessary as its already described here : https://developer.shopware.com/docs/products/extensions/b2b-components/employee-management/concept/entities-and-schema.html#role

Reply:
I think it still necessary, the role description in section entities-and-schema used for explain the meaning for table/entity swag_b2b_role in database. And the role in B2B-roles, explain more detail about how Role business logic work.